src/DataViews/PresetsDataViewTest.cpp

```diff
@@ -208,19 +208,20 @@ TEST_F(PresetsDataViewTest, CheckPresenceOfContextMenuEntries) {
 
   // Loadable preset
   EXPECT_THAT(FlattenContextMenuWithGrouping(view_.GetContextMenuWithGrouping(0, {0})),
-              testing::UnorderedElementsAre("Copy Selection", "Export to CSV", "Load Preset",
-                                            "Delete Preset"))
+              testing::ElementsAre("Load Preset", "Delete Preset", "Show in Explorer",
```

Comment:
I love this change, but it should belong to another PR where we change all of them in the file.

Reply:
We can check the order in this PR because we enforce the single selection in the presets tab. However, we support multi-selection in other data views, which means that there can be hundreds of combination cases for the menu content and so we cannot use `ElementsAre` to check the order.

Reply:
My bad, I saw UnorderedElementsAre in lines 175, 178 and supposed you could have changed it. I still think that this should be in another PR, but I'm not too strong about that.

For other dataviews (discussion for other PR), we can check instead than the current order of actions have the same order than the theoretical order (including all the actions), so if all actions are A - B - C - D - E ...., we don't have something like A - E - C in our array. Let me know if you want to discuss it, a single-for should be enough.

Reply:
Thanks for the suggestion! 

I have an idea about how to test the action orders. But I would like to do it together with the later refactor of the data views XD